tests/sklearn/test_sklearn_models.py

```diff
@@ -1363,25 +1357,26 @@ def test_input_support(
 
 
 @pytest.mark.parametrize("model_class, parameters", UNIQUE_MODELS_AND_DATASETS)
-@pytest.mark.parametrize(
-    "n_bits",
-    N_BITS_WEEKLY_ONLY_BUILDS + N_BITS_REGULAR_BUILDS,
-)
-def test_subfunctions(
+def test_inference_methods(
     model_class,
     parameters,
-    n_bits,
     load_data,
     is_weekly_option,
+    check_float_array_equal,
+    default_configuration,
     verbose=True,
 ):
-    """Test subfunctions."""
+    """Test inference methods."""
+    n_bits = min(N_BITS_REGULAR_BUILDS)
```

Comment:
this test only checks for result coherence between clear and simulation executions, so I believe we don't really need to use lots of different numbers of bits